ludwig/constants.py

```diff
@@ -282,6 +282,7 @@
 GENERATION = "generation"
 PROMPT = "prompt"
 ADAPTER = "adapter"
+PRETRAINED_WEIGHTS = "pretrained_weights"
```

Comment:
nit, might be more clear to call it `pretrained_adapter_weights` since pretrained weights also come from the model! So just to avoid confusion

Reply:
On it